contracts/Arbitrum_SpokePool.sol

```diff
@@ -0,0 +1,118 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./SpokePool.sol";
+import "./SpokePoolInterface.sol";
+
+interface StandardBridgeLike {
+    function outboundTransfer(
+        address _l1Token,
+        address _to,
+        uint256 _amount,
+        bytes calldata _data
+    ) external payable returns (bytes memory);
+}
+
+/**
+ * @notice AVM specific SpokePool.
+ * @dev Uses AVM cross-domain-enabled logic for access control.
+ */
+
+contract Arbitrum_SpokePool is SpokePoolInterface, SpokePool {
```

Comment:
this does not have any tests. shall we add that in this PR or a separate one? we also need to write OP tests, which can happen either at the same time, if we choose to wait on adding the tests here.

Reply:
Let's do separate PR